orca-loadtest/src/main/resources/orca-simulation.conf

```diff
@@ -0,0 +1,15 @@
+service.orca {
+  serviceUrl = "http://localhost:8082"
+
+  rampUpPeriod = 10
+  duration = 30
+
+  submitOrchestration {
+    rampUsersPerSec = 1
+    rampUsersTo = 3
+  }
+
+  setup {
```

Comment:
If you don't think you need it, I'd remove this - helps declutter things.  You can always add it back later.